cpu/nrf52/radio/nrf802154/nrf802154_radio.c

```diff
@@ -347,6 +349,21 @@ void nrf802154_on(void)
 
 }
 
+static void _timer_cb(void *arg, int chan)
+{
+    (void)arg;
+    (void)chan;
+    ieee802154_dev_t *dev = &nrf802154_hal_dev;
+
+    ack[0] = 5;
+    ack[1] = 0x02;
+    ack[2] = 0;
+    ack[3] = rxbuf[3];
```

Comment:
sequence number should be `ack[2]`

Reply:
yes, this includes frame length. I will add a macro for the magic number

Reply:
`sizeof(ack)` would be enough. There are already maces for the rest.